style.css

```diff
@@ -0,0 +1,77 @@
+.container{
+    width: 95vw;
+    min-height: 1500px;
+    background-color:#eee;
+    font-family: Arial, Helvetica, sans-serif;
+    color: black;
+}
+.menu{
+    width: 100%;
+    height: 60px;
+    line-height: 60px;
+}
+.title{
+    clear: both;
+    width: 100%;
+    height: 150px;
+    text-align: center;
+    
+}
+.list{
+    width: 100%;
+    height: 400px;
+    background-color: rgb(217, 16, 136);
+}
+.box{
+    width: 100%;
+    height: 200px;
+    background-color: rgb(56, 16, 217);
+}
+.footer{
+    width: 100%;
+    height: 60px;
+    background-color: rgb(11, 11, 11);
+}
+.menu .logo img{
+    margin-top: 20px;    
+}
+.logo{
+    float: left;
+    line-height: 60px;
+}
+.menu_items{
+    margin-left: 15%;
+    margin-right: 30px;    
+    display: block;
+    float: right;
+}
+.menu_items a{
+    margin-left: 100px;
+}
+.menu_items a:link{
+    text-decoration: none;
+    color: black;
+}
+.menu_items a:visited{
+    text-decoration: none;
+    color: black;
+}
+.menu_items a:hover{
+    text-decoration: none;
+    color: black;
+}
+.menu_items a:active{
+    color: black;
+    text-decoration: none;
+}
+.title input{
+    width: 250px;
+    height: 25px;
+}
+.title input[type=submit]{
+    width: 150px;
+    height: 30px;
+    font-weight: 600;
+    background-color:#f1c40f;
+    border: none;
+}
```

Comment:
It seems you forgot to use linters in the project. It doesn't pass linters verification. Please a least add linters.yml to your project and fix the linters errors. For further reading about linters, use the following link: https://github.com/microverseinc/linters-config/tree/master/html-css